vmm/src/config.rs

```diff
@@ -711,6 +716,12 @@ impl DiskConfig {
             .add("socket")
             .add("poll_queue")
             .add("id")
+            .add("bw_size")
+            .add("bw_one_time_burst")
+            .add("bw_refill_time")
+            .add("ops_size")
+            .add("ops_one_time_burst")
+            .add("ops_refill_time")
```

Comment:
I would think we could condense this down. The user wants to specify in bytes per second or operations per second. We should fix the the refill time and scale the size parameters accordingly. I think we could also create a reasonable burst value based on what the user has provided.